lib/tapioca/runtime/trackers/constant_definition.rb

```diff
@@ -34,7 +46,7 @@ module ConstantDefinition
         def self.files_for(klass)
           name = String === klass ? klass : name_of(klass)
           files = @class_files[name]
-          files || Set.new
+          files&.map(&:path)&.to_set || Set.new
```

Comment:
Could we instead do a `.fetch(name, [])` in the line above so that this line becomes:
```suggestion
          files.map(&:path).to_set
```